app/src/renderer/components/governance/LiProposal.vue

```diff
@@ -31,23 +31,24 @@
     </td>
     <td>{{ `#` + proposal.proposal_id }}</td>
     <td class="li-proposal__value yes">
-      {{ tally.yes || `--` }}
+      {{ yesPercentage || `--` }}
```

Comment:
I haven't double checked with data, but codewise seems the percentage functions already covers the `--`